docs/docs/demos/time-series-enrich.md

```diff
@@ -1,3 +1,63 @@
-# Time Series Enrich
+# Time Series Enrichment
 
-Demo 02
\ No newline at end of file
+The first demo, on [Simple Data Selection](simple-select), was a very
+simple example of a DBSP pipeline.  Let's look at a slightly more
+sophisticated example.
+
+Suppose we have two tables.  One of them, `transactions`, reports
+credit card transactions as they occur, with information such as the
+transaction date and time and the amount.  The other, `demographics`,
+is information associated with credit card numbers, such as the name
+and address of its user:
+
+```mermaid
+erDiagram
+   demographics {
+     float64 cc_num
```

Comment:
I always found it strange to use float for a credit card number

Reply:
this should be either a decimal or a string

Reply:
I changed it to a string.

Credit card numbers are 16 digits, which fit inside 64-bit integers, but they're ordinals instead of scalars so yeah a string seems reasonable.